hooks/commit-msg.sample

```diff
@@ -14,3 +14,25 @@ if [ -z "$EMAIL" ]; then
 fi
 
 git interpret-trailers --if-exists doNothing --trailer "Signed-off-by: $NAME <$EMAIL>" --in-place "$1"
+
+# Commits message validation
+Color_Off='\033[0m'
+BRed="\033[1;31m"         # Red
+BGreen="\033[1;32m"       # Green
+BYellow="\033[1;33m"      # Yellow
+BBlue="\033[1;34m"        # Blue
```

Comment:
Not strictly necessary and very likely to break. I appreciate the effort, but I'd rather the variables were clearer.